client/my-sites/plans/main.jsx

```diff
@@ -70,6 +74,20 @@ var Plans = React.createClass( {
 		);
 	},
 
+	redirectToDefault() {
+		page.redirect( paths.plans( this.props.selectedSite.slug ) );
+	},
+
+	renderNotice() {
```

Comment:
I guess we could but then why didn't we do that for other actions such as purchasing a free trial?